apps/emqx_ft/test/emqx_ft_responder_SUITE.erl

```diff
@@ -20,15 +20,19 @@
 -compile(nowarn_export_all).
 
 -include_lib("stdlib/include/assert.hrl").
+-include_lib("common_test/include/ct.hrl").
 
 all() -> emqx_common_test_helpers:all(?MODULE).
 
 init_per_suite(Config) ->
-    ok = emqx_common_test_helpers:start_apps([emqx_ft], emqx_ft_test_helpers:env_handler(Config)),
-    Config.
+    {ok, Apps} = application:ensure_all_started(gproc),
```

Comment:
Nit: why should we explicitly mock internals now?

Reply:
We shouldn't. Idea was: in order to test responder stuff, we don't actually need `emqx_ft` (and thus `emqx`, `ekka`, `mria`), even though it's formally part of it. Yet that means we now should now what its runtime dependencies are.

Nevertheless, it seems that cb0b0c6da29bf7e6bb03b5febf031ac8369a0048 is somewhat questionable, I'll probably revert it.